File: solutions/1054-complement-of-base-10-integer/solution.py
class Solution(object):
    def bitwiseComplement(self, n):
        """
        :type n: int
        :rtype: int
        """
        s=""
        for i in bin(n)[2:]:
            if i=='0':
                s+='1'
            else:
                s+='0'
        return int(s,2)

File: solutions/1054-complement-of-base-10-integer/solution.java
class Solution {
    public int bitwiseComplement(int n) {
        String s="";
        for(char i:Integer.toBinaryString(n).toCharArray())
            if(i=='0')
                s+='1';
            else
                s+='0';
        return Integer.parseInt(s,2);
    }
}
